<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QDialog" name="AdminWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>					 
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>554</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>791</width>
    <height>554</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>791</width>
    <height>554</height>
   </size>
  </property>							   		 
  <property name="windowTitle">
   <string>AdminWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="Information">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>501</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>用户信息</string>
    </property>
    <widget class="QTableView" name="StuView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>481</width>
       <height>491</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Detailed">
    <property name="geometry">
     <rect>	   
     <x>510</x>
     <y>10</y>
      <width>271</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>详细信息</string>
    </property>
    <widget class="QListWidget" name="DetailInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="DelButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AddButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>360</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TargetName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="SearchButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>420</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查找</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ExitButton">
     <property name="geometry">
      <rect>
       <x>139</x>
       <y>483</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ModButton">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>483</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>StuView</tabstop>
  <tabstop>DetailInfo</tabstop>
  <tabstop>DelButton</tabstop>
  <tabstop>AddButton</tabstop>
  <tabstop>TargetName</tabstop>
  <tabstop>SearchButton</tabstop>
  <tabstop>ModButton</tabstop>
  <tabstop>ExitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
